### PR TITLE
Missing files in Eigen3

### PR DESCRIPTION
**Pull request by dsendkowski**
I've missed some files from Eigen3.

### CODE REVIEW

**Comment by pyhalov** [comment]
Are they present in smaple-manifest?

**Comment by dsendkowski** [comment]
Yes, they are.

**Comment by pyhalov** [comment]
please, bump COMPONENT_REVISION

File: components/scientific/eigen3/Makefile [hunk 1]
```diff
@@ -15,6 +15,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=           eigen
 COMPONENT_VERSION=        3.3.3
+COMPONENT_REVISION =	1
```
Comment (pyhalov) [inline components/scientific/eigen3/Makefile]: please, align it with other code

**Comment by pyhalov** [comment]
I don't want to be a nerd, but look at it https://github.com/OpenIndiana/oi-userland/pull/3082/files :)

**Comment by dsendkowski** [comment]
Now, it should be aligned :)

**Pull request merged**